src/main/kotlin/org/jitsi/utils/JavaVersion.kt

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
We might want to add `@file:JvmName("JavaVersion")` or something to the file here to make it friendlier to call from Java